devicetypes/smartthings/zigbee-ceiling-fan-light.src/itm-fan-child.groovy

```diff
@@ -0,0 +1,85 @@
+/*
+ *  Copyright 2017 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ *  ZigBee Ceiling Fan Light
+ *
+ *  Author: SAMSUNG LED
+ *  Date: 2021-06-30
+ */
+
+import physicalgraph.zigbee.zcl.DataType
+import groovy.json.JsonOutput
+
+metadata {
+	definition(name: "ITM Fan Child", namespace: "SAMSUNG LED", author: "SAMSUNG LED", ocfDeviceType: "oic.d.fan") {
+		capability "Actuator"
+        	capability "Configuration"
+        	capability "Refresh"
+        	capability "Switch"
+		capability "Switch Level"
```

Comment:
Can you add some comment about `Switch Level` capability, that it's really needed for controlling fans in 3rd party APIs.

Reply:
Sure, If I don't add Switch Level capability, level control bar doesn't appears. Only switch button shows.

Reply:
I've meant including comment in the code.

Reply:
Got it! Thank you